src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -55,6 +61,32 @@ function transformEventStats(data: EventsStatsData, seriesName?: string): Series
   ];
 }
 
+function transformEventStatsSmoothed(data: Series[], seriesName?: string): Series[] {
+  const currentData = data[0].data;
+  const resultData: SeriesDataUnit[] = [];
+
+  // Start at 1 so we don't ge NaN from the movingAverage
+  for (let i = 1; i < currentData.length; i++) {
```

Comment:
If you're starting at 1 will the moving average data be one result smaller than the unsmoothed data?

Reply:
Oh it is, I'll make the 0th point match the unsmoothed data's 0th point :+1: